runtime/domain.c

```diff
@@ -917,7 +944,9 @@ CAMLprim value caml_domain_spawn(value callback, value mutex)
     /* failed */
     pthread_join(th, 0);
     free_domain_ml_values(p.ml_values);
-    caml_failwith("failed to allocate domain");
+    caml_failwith("failed to allocate domain. "
+      "You can set the 'Max_domains' OCAMLRUNPARAM parameter "
```

Comment:
Should this be "max_domains (d)". See https://github.com/ocaml/ocaml/pull/10955/files#diff-315666779650ff6271e537465a48cd484e1f4671fafbfaa2ed22207a0c0aa038R132.